### PR TITLE
Split Perf  Helix Workitem

### PR DESCRIPTION
**Pull request by ooooolivia**
- update crossgen perf job runkind to ```crossgen_scenarios```
- split perf jobs based on runkind in ```perf_job.yml```

### CODE REVIEW

**Comment by ooooolivia** [comment]
@billwert  @DrewScoggins Can you review this? tagging here as there's no inviting review feature accessible for me

File: eng/pipelines/coreclr/perf.yml [hunk 1]
```diff
@@ -73,7 +76,10 @@ jobs:
       testGroup: perf
       liveLibrariesBuildConfig: Release
       runtimeType: mono
+      projectFile: microbenchmarks.proj
+      runKind: micro
```
Comment (DrewScoggins) [inline eng/pipelines/coreclr/perf.yml:80]: This needs to be micro_mono

File: eng/pipelines/coreclr/perf.yml [hunk 2]
```diff
@@ -87,7 +93,10 @@ jobs:
         liveLibrariesBuildConfig: Release
         runtimeType: mono
         codeGenType: 'Interpreter'
+        projectFile: microbenchmarks.proj
+        runKind: micro
```
Comment (DrewScoggins) [inline eng/pipelines/coreclr/perf.yml:97]: This also needs to be micro_mono

File: eng/common/templates/steps/perf-send-to-helix.yml [hunk 3]
```diff
@@ -1,10 +1,11 @@
 # Please remember to update the documentation if you make changes to these parameters!
 parameters:
+  ProjectFile: ''                        # required -- project file that specifies the helix workitems
   HelixSource: 'pr/default'              # required -- sources must start with pr/, official/, prodcon/, or agent/
   HelixType: 'tests/default/'            # required -- Helix telemetry which identifies what type of data this is; should include "test" for clarity and must end in '/'
   HelixBuild: $(Build.BuildNumber)       # required -- the build number Helix will use to identify this -- automatically set to the AzDO build number
   HelixTargetQueues: ''                  # required -- semicolon delimited list of Helix queues to test on; see https://helix.dot.net/ for a list of queues
-  HelixAccessToken: ''                   # required -- access token to make Helix API requests; should be provided by the appropriate variable group
+  HelixAccessToken: ''                   # required -- access token to make Helix API requests; should be provided by the appropriate variable group
```
Comment (DrewScoggins) [inline eng/common/templates/steps/perf-send-to-helix.yml:8]: nit: Space at end of line

**Review by DrewScoggins — COMMENTED** [review]
Thanks for adding all of these comments as well.

**Comment by ooooolivia** [comment]
https://github.com/dotnet/runtime/issues/40034

**Comment by DrewScoggins** [comment]
Once those checks pass on re-run LGTM.

**Review by DrewScoggins — APPROVED** [review]

**Pull request merged**